packages/dbos-cloud/applications/register-app.ts

```diff
@@ -27,8 +27,8 @@ export async function registerApp(dbname: string, host: string): Promise<number>
       }
     );
     const uuid = register.data as string;
-    logger.info(`Successfully registered: ${appName}`);
     logger.info(`${appName} ID: ${uuid}`);
+    logger.info(`Successfully registered ${appName}!`);
```

Comment:
orthogonal: do we need to expose the app ID to users? (We use it in tests but we can work around that.)

Reply:
Unclear--it definitely shows up in Redshift and maybe Grafana.  Discussion topic for later.